plansys2_terminal/src/plansys2_terminal/Terminal.cpp

```diff
@@ -82,7 +83,7 @@ char * completion_generator(const char * text, int state)
   std::vector<std::string> vocabulary_set{"instance", "predicate", "function", "goal"};
   std::vector<std::string> vocabulary_get{"model", "problem", "domain", "plan"};
   std::vector<std::string> vocabulary_remove{"instance", "predicate", "function", "goal"};
-  std::vector<std::string> vocabulary_run{"action", "num_actions"};
+  std::vector<std::string> vocabulary_run{"action", "num_actions", "planfrom"};
```

Comment:
- Add also `help` to completion, with all the arguments
- `planfrom` is not bad (I prefect `plan-from`), but maybe `plan-file` is better.

Reply:
Sure, I've forgot!

Reply:
Fixed in the latest push!